components/team_members_dropdown/__snapshots__/team_members_dropdown.test.tsx.snap

```diff
@@ -0,0 +1,189 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
The source of the conflict from my last PR may be from this snap file. This file was gone. (Why?)